src/magnify/apps/core/models.py

```diff
@@ -430,7 +430,22 @@ def save(self, *args, **kwargs):
             self.reset_recurrence()
 
         super().save(*args, **kwargs)
-
+ 
+    def next_occurrence_from_today(self):
+        """
+        Returns the next occurrence of the meeting from today.
+        """
+        if self.start == timezone.now().date():
```

Comment:
First I think we need to add a "timezone" field to the model to be aware of what the user meant when s.he entered the date and time of the meeting? (cf google... it is set by default from your profile but you can modify it when scheduling your meeting...)

Then we will be able to compute the exact and aware datetime of the meeting in order to compare it with timezone.now()?

Reply:
maybe we should compare with start + 1/2*expected_duration to know if the meeting is passed or not :smile:

Reply:
I use the "timezone" object to pay attention to the user's time zone as advised by django. Maybe we could use datetimefield instead of start and start_time

Reply:
> maybe we should compare with start + 1/2*expected_duration to know if the meeting is passed or not 😄

Why not... I used start + expected_duration to know if the meeting is still active or not

Reply:
Yes, that's the only way to know the exact time we are talking about :thinking: I remember having a reason not to merge these 2 infos into one field but I forgot and now it seems stupid to me. So yes I think we have to do that!

Reply:
I will prototype something next week.

Reply:
Not exactly next week but I propose this fix: https://github.com/openfun/jitsi-magnify/pull/140

Reply:
Hello, thanks for the information. I will have a look to this as soon as possible